Create advanced-alarm-handling.adoc (#7426)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/alarms/advanced-alarm-handling.adoc

= Advanced Alarm Handling
:description: Advanced ways to handle alarms in {page-component-title}: advanced alarm handling.

Advanced Alarm Handling using Drools.

[[ga-Included-alarm-handling]]
== Included Alarm handling

In addition to the manual actions, it is possible to automate alarm handling with the use of https://www.drools.org/[Drools] scripts.

Within the `$\{OPENNMS_HOME}/etc/alarmd/drools-rules.d/alarmd.drl` file there is a default rule set of rules for handling alarm cleanup, 
clearing, and creating/updating tickets.

> Within the Within the `$\{OPENNMS_HOME}/etc/alarmd/drools-rules.d/situations.drl` file there is a default rule set for handling alarm cleanup, 
> clearing, and creating/updating tickets.

Reviewer: Wasn't Within the Within a crappy 90's prog band?
```suggestion
The `$\{OPENNMS_HOME}/etc/alarmd/drools-rules.d/situations.drl` file contains the default rules for the Situation lifecycle.
```

Reviewer: Also, no mid-sentence line breaks (as I recall).  Keep everything on one line.

> You can find some additional examples in the `$\{OPENNMS_HOME}/etc/examples/alarmd/drools-rules.d/` directory.

Reviewer: ```suggestion
Additional examples are available in `$\{OPENNMS_HOME}/etc/examples/alarmd/drools-rules.d/`.
```
